packages/strapi-plugin-i18n/admin/src/components/LocaleRow/index.js

```diff
@@ -15,7 +15,10 @@ const LocaleSettingsPage = ({ locale, onDelete, onEdit }) => {
   if (onEdit) {
     links.push({
       icon: (
-        <span aria-label={formatMessage({ id: getTrad('Settings.list.actions.edit') })}>
+        <span
+          aria-label={formatMessage({ id: getTrad('Settings.list.actions.edit') })}
+          style={locale.isDefault ? { marginRight: '-16px' } : undefined}
```

Comment:
Are we sure there's no other way to align the icons?

Reply:
TBH, I don't really know. All of this is hidden in buffet in IconLinks somehow and we don't have that much control over them

Reply:
I think it is because of the delete icon button. 
An empty button is rendered for the default locale so it creates a "shift" between icons.

![Capture d’écran 2021-03-05 à 12 39 49](https://user-images.githubusercontent.com/7756284/110110985-e59db980-7daf-11eb-8030-45d4871a7cf8.png)

You can remove this custom style and change the condition on the line 29:

```js
if (onDelete || !locale.isDefault) {
```

Reply:
Nice catch, thank for the suggestion 🙏🏻